src/react/src/components/DashboardClaimsListTableHeader.jsx

```diff
@@ -0,0 +1,123 @@
+import React, { useCallback } from 'react';
+import { func, string } from 'prop-types';
+import TableHead from '@material-ui/core/TableHead';
+import TableSortLabel from '@material-ui/core/TableSortLabel';
+import TableRow from '@material-ui/core/TableRow';
+import TableCell from '@material-ui/core/TableCell';
+
+const DashboardClaimsListTableHeaderStyles = Object.freeze({
+    firstTh: {
+        width: '80px',
+    },
+    secondTh: {
+        width: '175px',
+    },
+    thirdTh: {
+        width: '145px',
+    },
+    fourthTh: {
+        width: '90px',
+    },
+    sixthTh: {
+        width: '100px',
+    },
+});
+
+// Table header ids should match keys from BE
+const claimsListHeadCells = [
```

Comment:
As it a new changes, we could also use classes here instead of const.

Reply:
`const claimsListHeadCells` is not a class, it's an object with data that populated to the table header.